tests/conftest.py

```diff
@@ -69,6 +72,9 @@ def review():
                     "name": "the-answer-to-the-ultimate-question"
                 },
                 "spec": {
+                    "jobTemplate": {"spec": {"template": {
+                        "spec": {"containers": [{"image": "4.2"}]}
+                    }}},
```

Comment:
I'd prefer that we keep this review object valid from a Kubernetes standpoint. Can we just put this in the related test directly?